packages/components/src/MultiSelect/subcomponents/MultiSelectToggle/MultiSelectToggle.tsx

```diff
@@ -77,9 +79,14 @@ export const MultiSelectToggle = forwardRef<
                   // This stops the underlying toggle collapsing the popover when interactive with Tags
                   // eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions, jsx-a11y/click-events-have-key-events
                   <li key={value} onClick={e => e.stopPropagation()}>
-                    <Tag classNameOverride={styles.selectedItemTag}>
+                    <RemovableTag
+                      removeButtonProps={{
+                        ariaLabel: "Remove option",
+                        onClick: () => deselectOption(value),
```

Comment:
As discussed in [Slack](https://cultureamp.slack.com/archives/C05D8V4GALX/p1698373147766779), we're going to stick with the convention where props will follow the `onEvent` pattern, as it is more scalable and gives control to the consumer without restricting api (any refactors would not result in a huge change either).

So something on the line of `onRemoveOption` would suffice, and the deselecting functionality handled by the consuming component (which you've done).